jp/environment.json

```diff
@@ -0,0 +1,14 @@
+{
+    "development": {
+        "infragisticsBaseUrl": "https://staging.infragistics.com",
+        "dvApiBaseUrl": "https://staging.infragistics.com"
+    },
+    "staging": {
+        "infragisticsBaseUrl": "https://staging.infragistics.com",
```

Comment:
The JP staging domain is little bit different - 
![image](https://user-images.githubusercontent.com/25771040/111990382-e78baa80-8b1b-11eb-86fc-7e8082ac247c.png)